src/components/Header/Header.test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import Header from "./Header";
+import StoreLogo from "./StoreLogo";
+import logoImage from "../../assets/images/store-logo.png";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<Header />", () => {
+  it("should render the Storelogo component", () => {
+    const header = shallow(<Header />);
+    const element = header.find(StoreLogo);
+
+    expect(element.name()).toBe("StoreLogo");
```

Comment:
Rather than testing if the component name was correct, the bonus refers to the name of your Ecommerce. For example, in your case your Ecommerce name is `Furniture` from what I saw. You are testing only the image, but you're missing the actual name of the Ecommerce. Other than that, good that you have added the bonus.